themes/default/scripts/admin.js

```diff
@@ -995,7 +995,7 @@ function select_in_category(operation, brd_list)
  * Server Settings > Caching, toggles input fields on/off as appropriate for
  * a given cache engine selection
  */
-$(document).ready(function() {
+$(function() {
 	$('#cache_accelerator').change(function() {
```

Comment:
Is this now recommended for 3.0? 

I know $(function() is the shorthand for $(document).ready, although the later has better readability IMO ... that said  I'm fine with either and less typing is always better.  Just curious if this was another recommended 3.0 change that I missed 😇 they are awesome at hiding little depreciated things in the docs.